### PR TITLE
Week 1 homework akbar fixed commented mistakes

### PR DESCRIPTION
**Pull request by Akbar1002**

### CODE REVIEW

File: Week1/js-exercises/ex5-catWalk.js [hunk 1]
```diff
@@ -10,4 +10,33 @@
  5. When the cat reaches the right - hand of the screen, restart them at the left hand side("0px").So they should keep walking from left to right across the screen, forever and ever.
```
Comment (cgduncan7) [inline Week1/js-exercises/ex5-catWalk.js:10]: `git add -A` ☹️ 
`git add Week2` 😄

File: Week2/js-exercises/ex1-oddOnesOut.js [hunk 2]
```diff
@@ -7,15 +7,19 @@
...
+//   }
+//   return newNumbers;
+// }
 
 const myNumbers = [1, 2, 3, 4];
-console.log(doubleEvenNumbers(myNumbers)); // Logs "[4, 8]" to the console
\ No newline at end of file
+// console.log(doubleEvenNumbers(myNumbers)); // Logs "[4, 8]" to the console
+
+const evenNum = myNumbers.filter((num) => num % 2 === 0).map((element) => element * 2);
```
Comment (cgduncan7) [inline Week2/js-exercises/ex1-oddOnesOut.js:23]: You didn't exactly rewrite the function...

File: Week2/js-exercises/ex2-whatsYourMondayWorth.js [hunk 3]
```diff
@@ -11,12 +11,26 @@
...
   // put your code in here, the function does returns a euro formatted string
+  const time = tasks.map(function (activity) {
+    return activity.duration;
+  });
+  const totalTime = mondayTasks.reduce(function (total, time) {
+    return total + time.duration;
+  }, 0);
+
+  const hour = totalTime / 60;
+  const hourworth = (hour * hourlyRate).toFixed(2);
```
Comment (cgduncan7) [inline Week2/js-exercises/ex2-whatsYourMondayWorth.js:24]: 👍

File: Week2/js-exercises/ex3-lemonAllergy.js [hunk 4]
```diff
@@ -11,11 +11,16 @@
 
  */
 
-
 function takeOutLemons(basket) {
-  // your code goes in here. The output is a string 
+  // your code goes in here. The output is a string
+  const withoutLemon = basket.filter(function (fruit) {
```
Comment (cgduncan7) [inline Week2/js-exercises/ex3-lemonAllergy.js:16]: This is not how filter should be used. The first argument for `filter` is a function which should return a boolean value.

File: Week2/js-exercises/ex5-myFavoriteHobbies.js [hunk 5]
```diff
@@ -8,14 +8,23 @@
  Put the list items in an unordered list
  */
 
-function createHTMLList(arr) {
-  // your code goes in here
-}
+// function createHTMLList(arr) {
+//   // your code goes in here
```
Comment (cgduncan7) [inline Week2/js-exercises/ex5-myFavoriteHobbies.js:12]: your code goes here 🤔

**Review by cgduncan7 — CHANGES_REQUESTED** [review]
No project code was submitted. Please do this.

The rest looks okay. When the exercises say `code goes here`, should write the code there and not change existing lines.

**Comment by Akbar1002** [comment]
> No project code was submitted. Please do this.
> 
> The rest looks okay. When the exercises say `code goes here`, should write the code there and not change existing lines.

File: Week2/js-exercises/ex1-oddOnesOut.js (hunk 2 above)
Comment (Akbar1002) [reply Week2/js-exercises/ex1-oddOnesOut.js:23]: hi, I think I messed up some files here. this is supposed to be week1 (fixed files )not for week 2 so the project is "Quotes generator" which is already inside , but it created a new pull request I don't know why, and the week 2 files with it. 
 I don't know how to fix it now , but my project for week2 is in another pull request below , week 2 akbar . if you would like to check it .

**Pull request closed without merging**